packages/core/lib/config/handler.js

```diff
@@ -39,11 +40,13 @@ const resolveConfig = async (
   let configs = []
   if (Array.isArray(config.extends) && config.extends.length > 0) {
     config.extends = extendsResolver(config.extends, context)
-    configs = await Promise.all(
-      config.extends.map((configPath) =>
-        resolveConfigFile(configPath, depth + 1),
-      ),
-    )
+    if (Array.isArray(config.extends)) {
```

Comment:
What if it isn't? Maybe some log warning?
Also, this entire block is strange. It checks the array first, then calls `extendsResolver` and needs to check again?

Reply:
I needed to add this check before I introduce better typings.
It's now able to correctly evaluate the return type as an array, so I removed the check.

Thanks for catching this!